routing/routing_algorithm.cpp

```diff
@@ -19,7 +19,12 @@ double constexpr KMPH2MPS = 1000.0 / (60 * 60);
 inline double TimeBetweenSec(Junction const & j1, Junction const & j2, double speedMPS)
 {
   ASSERT(speedMPS > 0.0, ());
-  return MercatorBounds::DistanceOnEarth(j1.GetPoint(), j2.GetPoint()) / speedMPS;
+  ASSERT_NOT_EQUAL(j1.GetAltitude(), feature::kInvalidAltitude, ());
+  ASSERT_NOT_EQUAL(j2.GetAltitude(), feature::kInvalidAltitude, ());
+
+  double const distanceM = MercatorBounds::DistanceOnEarth(j1.GetPoint(), j2.GetPoint());
+  feature::TAltitude const altidudeDiffM = j2.GetAltitude() - j1.GetAltitude();
```

Comment:
I did not got the idea. TAltitude is defined as:
```c++
using TAltitude = int16_t;
```
The limits are -32,768 .. 32,767;
So I'll get integer overflow for height difference more 32K. Right?